crates/bws/src/command/mod.rs

```diff
@@ -44,16 +44,33 @@ pub(crate) fn config(
         config::delete_profile(config_file.as_deref(), profile)?;
         println!("Profile deleted successfully!");
     } else {
-        let (name, value) = match (name, value) {
+        let (name, mut value) = match (name, value) {
```

Comment:
I feel like making this mutable when it's just changing for this case could be improved, for example we could just match against the specific key:
```rust
 let (name, value) = match (name, value) {
            (None, None) => bail!("Missing `name` and `value`"),
            (None, Some(_)) => bail!("Missing `value`"),
            (Some(_), None) => bail!("Missing `name`"),
            (Some(ProfileKey::state_opt_out), Some(value)) => {
                // Do the thing
            }
            (Some(name), Some(value)) => (name, value),
        };
```

Reply:
Fixed in https://github.com/bitwarden/sdk/pull/930/commits/523b2dc0ca7c56f51ac279e3ea33f2f242a14fb5